### PR TITLE
add: delete option for facility bed type

### LINKED ISSUE #2833: Bed Capacity in Facility page : Unable to delete
once we add a bed capacity, we have no option to delete it

### PR DESCRIPTION
**Pull request by kunatastic**
fixes #2833 

image

### CODE REVIEW

**Review by skks1212 — APPROVED** [review]

**Review by cp-Coder — APPROVED** [review]

**Comment by nihal467** [comment]
@kunatastic once i click delete, the button get vanished

**Comment by kunatastic** [comment]
@nihal467 This pr solves the delete option for bed type. and for doctor type separate pr was created #2953 and the screenshot that you have attached is of deleting doctors. 

Can you test this PR again.

**Comment by nihal467** [comment]
test approved

File: src/Components/Facility/BedTypeCard.tsx [hunk 1]
```diff
@@ -1,17 +1,54 @@
...
 const BedTypeCard = (props: BedTypeProps) => {
   const bed = BED_TYPES.find((i) => i.id === props.room_type);
   const roomType = bed ? bed.text : "Unknown";
+
+  const dispatchAction: any = useDispatch();
+  const [openDeleteDialog, setOpenDeleteDialog] = useState(false);
+
+  const handleDeleteSubmit = async () => {
+    if (props.room_type) {
+      console.log(props);
```
Comment (khavinshankar) [inline src/Components/Facility/BedTypeCard.tsx]:
```suggestion
```

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**